Tribler/Test/test_gui_dialogs.py

```diff
@@ -176,18 +175,26 @@ def disable_dispersy():
         self.startTest(disable_dispersy)
 
     def test_confirmationdialog(self):
```

Comment:
It hangs with the current structure (build times out). However, seeing now that my current code suddenly hangs in `test_remove_dialog` as well.